include/bootloader/core/ids.h

```diff
@@ -128,6 +128,8 @@ typedef enum {
     can_errorcode_timeout = 0x6,
     can_errorcode_estop_detected = 0x7,
     can_errorcode_collision_detected = 0x8,
+    can_errorcode_labware_dropped = 0x9,
```

Comment:
this shouldn't be in here I think

Reply:
this was generated with update-headers from the changes already being in monorepo